app/src/main/java/cm/aptoide/pt/home/apps/AppsManager.java

```diff
@@ -219,7 +243,7 @@ public Completable updateApp(App app) {
               }
             })
             .map(__ -> download))
-        .flatMapCompletable(download -> installManager.install(download))
+        .flatMapCompletable(download -> installManager.install(download, isAppcUpdate))
```

Comment:
wrong flag being used here? I don't think you wanted to pass this flag because you are passing the isAppcUpdate and the variable is forceDefaultInstall